src/hooks/useCurrencyUSDPrice.ts

```diff
@@ -0,0 +1,9 @@
+import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
```

Comment:
isn't this `useUSDValue`? 

https://github.com/Uniswap/interface/blob/main/src/hooks/useUSDCPrice.ts#L58

Reply:
my god yes it is

Reply:
thank u 🤯